### PR TITLE
Add note on disk requirements for etcd (#1589)

### PR DESCRIPTION
**Pull request by songxianj**

### CODE REVIEW

File: site/en/getstarted/prerequisite-helm.md [hunk 1]
```diff
@@ -11,28 +11,23 @@ summary: Learn the necessary preparations before installing Milvus.
 
 Before you install Milvus, check your hardware and software to see if they meet the requirements.
 
-{{tab}}
```
Comment (claireyuw) [inline site/en/getstarted/prerequisite-helm.md:14]: Why are we removing this? This syntax renders as below:
image

**Review by claireyuw — CHANGES_REQUESTED** [review]

File: site/en/getstarted/prerequisite-helm.md (hunk 1 above)
Comment (songxianj) [reply site/en/getstarted/prerequisite-helm.md:14]: This mistake has been rectified along with other errors and clarity has been improved. Disk requirements have also been added in prerequisite-docker.md.

File: site/en/getstarted/prerequisite-docker.md [hunk 2]
```diff
@@ -30,13 +30,27 @@ Before you install Milvus, check your hardware and software to see if they meet
 | Linux platforms            | <ul><li>Docker 19.03 or later</li><li>Docker Compose 1.25.1 or later</li></ul> | See [Install Docker Engine](https://docs.docker.com/engine/install/) and [Install Docker Compose](https://docs.docker.com/compose/install/) for more information. |
 | Windows with WSL 2 enabled | Docker Desktop                                               | We recommend that you store source code and other data bind-mounted into Linux containers in the Linux file system instead of the Windows file system.<br/>See [Install Docker Desktop on Windows with WSL 2 backend](https://docs.docker.com/desktop/windows/install/#wsl-2-backend) for more information. |
 
-| Software | Version                       |
-| -------- | ----------------------------- |
-| etcd     | 3.5.0                         |
+| Software | Version                       | Note |
+| -------- | ----------------------------- | ---- |
+| etcd     | 3.5.0                         |  See [additional disk requirements](###-additional-disk-requirements). |
```
Comment (claireyuw) [inline site/en/getstarted/prerequisite-docker.md:35]: You need to add a pipe at the end of the other two rows below to form a complete table. 
See https://www.markdownguide.org/extended-syntax/#tables

File: site/en/getstarted/prerequisite-helm.md [hunk 3]
```diff
@@ -20,30 +20,41 @@ Before you install Milvus, check your hardware and software to see if they meet
...
 | Operating system | Software                                                     | Note                                                         |
 | ---------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
 | Linux platforms  | <ul><li>Kubernetes 1.16 or later</li><li>kubectl</li><li>Helm 3.0.0 or later</li><li>minikube (for Milvus standalone)</li><li>Docker 19.03 or later (for Milvus standalone)</li></ul> | See [Helm Docs](https://helm.sh/docs/) for more information. |
 
-| Software | Version                       |
-| -------- | ----------------------------- |
-| etcd     | 3.5.0                         |
+| Software | Version                       | Note |
+| -------- | ----------------------------- | ---- |
+| etcd     | 3.5.0                         |  See [additional disk requirements](###-additional-disk-requirements). |
```
Comment (claireyuw) [inline site/en/getstarted/prerequisite-helm.md:39]: You need to add a pipe at the end of the other two rows below to form a complete table.
See https://www.markdownguide.org/extended-syntax/#tables

File: site/en/getstarted/prerequisite-docker.md (hunk 2 above)
Comment (songxianj) [reply site/en/getstarted/prerequisite-docker.md:35]: Error rectified.

File: site/en/getstarted/prerequisite-helm.md (hunk 3 above)
Comment (songxianj) [reply site/en/getstarted/prerequisite-helm.md:39]: Error rectified.

**Pull request merged**